src/visitors/components/FeedbackButtons.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { Row, Col } from 'react-flexbox-grid';
+import { PrimaryButtonNoFill } from '../../shared/components/form/base';
+import confused from '../../shared/assets/icons/faces/confused.svg';
+import happy from '../../shared/assets/icons/faces/happy.svg';
+import sad from '../../shared/assets/icons/faces/sad.svg';
+
+
+const FeedbackButton = styled(PrimaryButtonNoFill)`
```

Comment:
Can this refactoring be pulled out into a separate PR?

Reply:
See #663